bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/BusyIndicator.java

```diff
@@ -70,6 +70,36 @@ public static void showWhile(Display display, Runnable runnable) {
 		}
 	}
 
+	/**
+	 * If called from a {@link Display} thread waits for the given
+	 * <code>Future</code> to complete providing busy feedback using the
+	 * busy indicator. If called from a non {@link Display} the method return
```

Comment:
The "non" all by itself is not a word.  It should be separated with a - from the thing it's "non"ing.  Also, Display should be followed by a coma.  And it should be the method return**s**, i.e., with an "s".